js/apps/admin-ui/src/user/user-credentials/ResetCredentialDialog.tsx

```diff
@@ -56,6 +56,7 @@ export const ResetCredentialDialog = ({
         id: userId,
         actions,
         lifespan,
+        clientId: "security-admin-console",
```

Comment:
Since the realm passed into this is based on the selected realm, could it be that this call fails if you run it from another realm than where the client resides?

Reply:
I just wanted to say that the `clientId` of the admin console in any realm is always `security-admin-console`, so I think that it will work in any realm. I did a quick test and it works in my env. And one more note, after this [PR](https://github.com/keycloak/keycloak/pull/20961) wouldn't be better using `environment.clientId` instead of hardcoded?